### PR TITLE
Datepicker

### LINKED ISSUE #124: Change datetime inputs to datepicker component
**Description**
To improve usability and accessibility, we have decided to change datetime inputs to use a datepicker instead.

We have landed on the following library: [react-datepicker](https://www.npmjs.com/package/react-datepicker), as it is simple, features date + time, and is accessible + customizable.

As design options might be limited, implementation will probably end up deciding design, so collaboration with @jens-concordium is important.

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Replaces timestamp input with proper datepicker.

Closes #124 

## Changes
- Deleted timestamp input
- Added datepicker component
- Small layout adaptions where needed to make UI behave with new component

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: app/components/Form/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            {label && <Label className="mB5">{label}</Label>}
+            <ReactDatePicker
+                selected={value}
+                onChange={onChange}
+                onBlur={onBlur}
+                showTimeSelect
+                timeCaption="Time"
+                timeFormat="HH:mm"
+                timeIntervals={15}
+                dateFormat="dd-MM-yyyy 'at' HH:mm:ss"
```
Comment (shjortConcordium) [inline app/components/Form/DatePicker/DatePicker.tsx:71]: When we display dates, we usually use `yyyy-MM-dd`, maybe it should also be like that here?

Comment (soerenbf) [reply app/components/Form/DatePicker/DatePicker.tsx:71]: I agree, they should definitely be aligned.. but doesn't it make more sense to display it with year last, as dates are normally displayed?

File: app/utils/timeHelpers.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { setHours, setMinutes, setSeconds, setMilliseconds } from 'date-fns';
```
Comment (shjortConcordium) [inline app/utils/timeHelpers.ts:1]: Can we just use functions from a transitive dependency? :thinking:

File: app/utils/timeHelpers.ts [hunk 3]
```diff
@@ -241,3 +242,20 @@ export function subtractHours(hours: number, date: Date) {
 export function parseTime(timeStamp: string | bigint, unit?: TimeStampUnit) {
     return getFormattedDateString(dateFromTimeStamp(timeStamp, unit));
 }
+
+const stripTime = (date: Date) =>
+    setHours(setMinutes(setSeconds(setMilliseconds(date, 0), 0), 0), 0);
```
Comment (shjortConcordium) [inline app/utils/timeHelpers.ts:247]: wouldn't it be easier to just clone and clone.setHours(0,0,0,0)?

File: app/components/Form/DatePicker/DatePicker.tsx (hunk 1 above)
Comment (shjortConcordium) [reply app/components/Form/DatePicker/DatePicker.tsx:71]: Is ISO 8601 not the norm?

Comment (shjortConcordium) [reply app/components/Form/DatePicker/DatePicker.tsx:71]: is ISO 8601 not the norm?

File: app/utils/timeHelpers.ts (hunk 2 above)
Comment (soerenbf) [reply app/utils/timeHelpers.ts:1]: You can, but it's obviously pretty bad to do.. I'll make sure they're installed.

File: app/components/Form/DatePicker/DatePicker.tsx (hunk 1 above)
Comment (soerenbf) [reply app/components/Form/DatePicker/DatePicker.tsx:71]: seems like it is 😅 I'll adapt.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**